package.json

```diff
@@ -13,59 +14,9 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@auth/prisma-adapter": "^1.0.2",
-    "@cms/packages": "workspace:^",
-    "@nextui-org/react": "^2.2.9",
-    "@radix-ui/react-accordion": "^1.1.2",
-    "@radix-ui/react-alert-dialog": "^1.0.5",
-    "@radix-ui/react-avatar": "^1.0.4",
-    "@radix-ui/react-dialog": "^1.0.5",
-    "@radix-ui/react-label": "^2.0.2",
-    "@radix-ui/react-popover": "^1.0.7",
-    "@radix-ui/react-scroll-area": "^1.0.5",
-    "@radix-ui/react-select": "^2.0.0",
-    "@radix-ui/react-slot": "^1.0.2",
-    "@radix-ui/react-tabs": "^1.0.4",
-    "@radix-ui/react-toggle": "^1.0.3",
-    "@radix-ui/react-tooltip": "^1.0.7",
-    "@types/react": "18.2.21",
-    "@types/react-dom": "18.2.7",
-    "autoprefixer": "10.4.15",
-    "class-variance-authority": "^0.7.0",
-    "clsx": "^2.0.0",
-    "cmdk": "^0.2.0",
-    "eslint": "8.48.0",
-    "eslint-config-next": "13.4.19",
-    "framer-motion": "^10.16.4",
-    "jss": "^10.10.0",
-    "jss-plugin-camel-case": "^10.10.0",
-    "jss-plugin-expand": "^10.10.0",
-    "jss-plugin-global": "^10.10.0",
-    "jss-plugin-nested": "^10.10.0",
-    "jss-plugin-props-sort": "^10.10.0",
-    "jss-preset-default": "^10.10.0",
-    "lucide-react": "^0.303.0",
-    "nanoid": "^5.0.1",
-    "next": "14.0.4",
-    "next-auth": "^4.23.1",
-    "next-themes": "^0.2.1",
-    "postcss": "8.4.28",
-    "react": "18.2.0",
-    "react-best-gradient-color-picker": "^2.3.5",
-    "react-dom": "18.2.0",
-    "react-icons": "^4.11.0",
-    "sass": "^1.69.5",
-    "slate": "^0.100.0",
-    "slate-react": "^0.100.0",
-    "slugify": "^1.6.6",
-    "sonner": "^1.3.1",
-    "tailwind-merge": "^1.14.0",
-    "tailwindcss": "3.3.3",
-    "tailwindcss-animate": "^1.0.7",
-    "ts-node": "^10.9.1",
-    "typescript": "5.2.2",
-    "zod": "^3.22.2",
-    "zustand": "^4.4.3"
+    "prettier": "^3.2.4",
+    "eslint": "^8.56.0",
+    "turbo": "^1.11.3"
```

Comment:
I installed `turbo` because it made it way faster to get work done. 😄

Reply:
Hmm okay, so you need to have turbo installed, okay!